content/about.md

```diff
@@ -26,7 +26,7 @@ the Hayden at his personal email address
 ## Disclaimer
 
 Rust for Lunch is not associated with or endorsed by either the
```

Comment:
If we use "nor", then it should be preceeded by "neither" (not either).

```suggestion
Rust for Lunch is not associated with or endorsed by neither the
```

Reply:
Right you are, my brain read that part "correctly" but when it got to the "or" it was like nope.